### ISSUE TITLE
Primitive's DeepCopy() method does not actually make a deep copy

### ISSUE THREAD

**Issue by dc7303** [yorkie-team/yorkie#66]
While analyzing the code, I noticed that the `DeepCopy()` method of the `Primitive` structure is not actually doing a deep copy.
Is this intended code because there is no problem with this?
Or can problems arise?

If we need to modify this code, can I pull request it?

Here I attach the test code that I ran temporarily.

```go
package json_test

import (
	"fmt"
	"testing"

	"github.com/stretchr/testify/assert"
	"github.com/yorkie-team/yorkie/pkg/document/json"
	"github.com/yorkie-team/yorkie/pkg/document/time"
)

func TestPrimitive(t *testing.T) {
	t.Run("DeepCopy test", func(t *testing.T) {
		origin := json.NewPrimitive(10, time.NewTicket(0, 0, time.InitialActorID))
		copied := origin.DeepCopy()

		originAddr := fmt.Sprintf("%p", origin)
		copiedAddr := fmt.Sprintf("%p", copied)
		assert.NotEqual(t, originAddr, copiedAddr)
	})
}
```

```bash
--- FAIL: TestPrimitive (0.00s)
    --- FAIL: TestPrimitive/DeepCopy_test (0.00s)
        primitive_test.go:20: 
                Error Trace:    primitive_test.go:20
                Error:          Should not be: "0xc00010db00"
                Test:           TestPrimitive/DeepCopy_test
FAIL
FAIL    github.com/yorkie-team/yorkie/pkg/document/json 0.401s
FAIL
```

**Comment by hackerwins** [comment]
@dc7303 Thanks for reporting.

Primitive type is implemented as immutable whose properties do not change in the early days of development. This was intended to improve speed when creating clones of Documents.

Since DeepCopy is only used to create a clone of a document, it won't be a problem right now but when deleting an instance of Primitive, it seems that it cannot be considered as immutable because it changes properties.

I think it would be good to send a PR.

**Comment by hackerwins** [comment]
This was solved by #67.
